cli/exchange/node.go

```diff
@@ -465,3 +535,474 @@ func NodeListErrors(org string, credToUse string, node string, long bool) {
 		fmt.Printf("%s\n", jsonBytes)
 	}
 }
+
+func ValidateUserInput(org string, credToUse string, userInput policy.UserInput, topLevelServices []ServiceReference, userInputsMap map[string]policy.UserInput, patternUserInputsMap map[string]policy.UserInput, userInputWithEmptyDefaultValueMap map[string][]exchange.UserInput) (bool, string, error) {
```

Comment:
Somewhere we need checks to verify that the input userinput structure is valid. If you pass in a valid JSON struct that doesnt have all the right fields, you will get wierd errors:
for input {"userInput":[{"foo":"bar"}]}
you get:
Start validating userinput , , ,
Error: bad HTTP code 405 from GET http://exchange-api:8080/v1/orgs//services?url=&arch=, output:

Reply:
Will return `Error: ServiceOrgid and ServiceUrl need to be set in userinput` in new pr